build/testdata/bundles/terraform/terraform/outputs.tf

```diff
@@ -18,3 +18,7 @@ output "boolean_var" {
 output "number_var" {
   value = var.number_var
 }
+
+output "json_encode_html" {
+  value = "hello&world?@test#$><"
```

Comment:
Can we follow the existing pattern for the other test cases, defining the value as a variable. Also it seems like the variables value are not currently used in the test-cli.sh. It feels confusing that's not used. If we are going to define it, I think we should use it in our tests.